package.json

```diff
@@ -5,41 +5,39 @@
   "author": "",
   "main": "app.js",
   "license": "ISC",
-  "jest": {
-    "verbose": true,
-    "bail": true,
-    "setupTestFrameworkScriptFile": "<rootDir>tests/setupEnzyme.js"
-  },
   "engines": {
     "node": ">=6.13.0"
   },
   "scripts": {
     "start": "nodemon server/index.js",
-    "test": "jest --coverage",
+    "test": "jest",
     "build": "webpack -d --watch",
     "seed": "node database/seed.js",
     "schema": "mysql -u root < schema.sql"
   },
   "dependencies": {
     "axios": "^0.18.0",
-    "babel-jest": "^23.6.0",
     "body-parser": "^1.18.3",
     "express": "^4.16.4",
     "faker": "^4.1.0",
     "mysql": "^2.16.0",
     "mysql2": "^1.6.1",
     "react": "^16.6.0",
     "react-dom": "^16.6.0",
-    "sequelize": "^4.41.0"
+    "sequelize": "^4.41.0",
+    "styled-components": "^4.0.3"
   },
   "devDependencies": {
     "@babel/core": "^7.1.2",
     "@babel/preset-env": "^7.1.0",
     "@babel/preset-react": "^7.0.0",
-    "babel-core": "^7.0.0-bridge.0",
     "babel-loader": "^8.0.4",
+    "babel-plugin-styled-components": "^1.8.0",
+<<<<<<< HEAD
     "enzyme": "^3.7.0",
     "enzyme-adapter-react-16": "^1.6.0",
+=======
+>>>>>>> 3e89b6e8121b66daea8c66f6f324e58695761ba8
     "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor",
```

Comment:
Unresolved file conflicts?

Reply:
Yes this is part of a bigger problem I'm having with git right now. Will try to resolve once I can get this PR to merge with my master.